packages/documentation/docs/user-guide/installation/quick-install.md

```
@@ -0,0 +1,172 @@
+  #   restart: always
+  #   command: yarn start -host core -port 3000 -id inewsGateway0
+  #   networks:
+  #     - sofie
+  #   depends_on:
+  #     - core
+
+  # rundown-editor:
+  #   image: ghcr.io/superflytv/sofie-automation-rundown-editor:v2.2.4
+  #   restart: always
+  #   ports:
+  #   	- '3010:3010'
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 7

(MD010, no-hard-tabs)